src/main/resources/templates/application/proceedings-remove.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'cases'},breadcrumbs=~{::#breadcrumbs})}" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>LAA CCMS - Start new application</title>
+</head>
+<body>
+<div class="govuk-width-container">
+  <div id="breadcrumbs">
+    <a th:href="@{/application/application/proceedings-and-costs}" class="govuk-back-link">Cancel and return to proceeding and costs</a>
+  </div>
+  <div id="main-content">
+    <form action="#" th:action="@{/application/proceedings/{proceedingId}/remove(proceedingId=${proceedingId})}" method="post">
+      <h1 class="govuk-heading-l">
+        Delete scope limitation
+      </h1>
+      <p class="govuk-body-l">
+        Are you sure you want to delete this proceeding? Please click 'Confirm' to delete
+        this information otherwise click 'Cancel and return to proceeding and costs' to go back.
```

Comment:
Please click 'Confirm' to delete this information**.** Otherwise**,** click 'Cancel and return to proceeding**s** and costs' to go back.

Reply:
amended text